src/Core/Banshee.WebBrowser/Banshee.WebSource/WebView.cs

```diff
@@ -119,17 +144,17 @@ public void FullReload ()
             // messages, since we do the streaming of previews natively.
             if (FixupJavascriptUrl != null && !fixup_javascript_fetched) {
                 fixup_javascript_fetched = true;
-                new Hyena.Downloader.HttpStringDownloader () {
+                new HttpStringDownloader
+                {
                     Uri = new Uri (FixupJavascriptUrl),
-                    Finished = (d) => {
-                        if (d.State.Success) {
-                            fixup_javascript = d.Content;
-                        }
+                    Finished = d => {
+                        if (d.State.Success) fixup_javascript = d.Content;
                         LoadHome ();
                     },
-                    AcceptContentTypes = new [] { "text/javascript" }
+                    AcceptContentTypes = new[] {"text/javascript"}
```

Comment:
not in the guidelines but I'd prefer it with at least the space between the curlies and the elements

Reply:
Sure thing. Another change for the R# PR. :+1: